scripts/just.config.js

```diff
@@ -41,38 +40,35 @@ module.exports = function preset() {
   task('copy', copy);
   task('jest', jest);
   task('jest-watch', jestWatch);
-  task('sass', sass);
   task('ts:commonjs', ts.commonjs);
   task('ts:esm', ts.esm);
-  task('ts:amd', ts.amd);
   task('eslint', eslint);
   task('ts:commonjs-only', ts.commonjsOnly);
   task('webpack', webpack);
-  task('metroPack', metroPackTask(argv()['bundleName']));
+  task('metroPack', () => metroPackTask(argv()['bundleName']));
   task('webpack-dev-server', webpackDevServer);
   task('verify-api-extractor', verifyApiExtractor);
   task('update-api-extractor', updateApiExtractor);
   task('prettier', prettier);
   task('bundle-size-collect', bundleSizeCollect);
+  task('check-for-modified-files', checkForModifiedFiles);
   task('generate-version-files', generateVersionFiles);
   task('generate-package-manifest', generatePackageManifestTask);
-  task('ts', () => {
-    return argv().commonjs
-      ? 'ts:commonjs-only'
-      : parallel('ts:commonjs', 'ts:esm', condition('ts:amd', () => argv().production && !argv().min));
-  });
+  task(
+    'ts',
+    series(condition('ts:commonjs-only', () => argv().commonjs), condition(parallel('ts:commonjs', 'ts:esm'), () => !argv().commonjs))
+  );
 
-  task('validate', fs.existsSync(path.join(process.cwd(), 'jest.config.js')) ? series('eslint', 'jest') : 'eslint');
+  task('validate', fs.existsSync(path.join(process.cwd(), 'jest.config.js')) ? parallel('eslint', 'jest') : 'eslint');
   task('code-style', series('prettier', 'eslint'));
-  task('update-api', series('clean', 'copy', 'sass', 'ts', 'update-api-extractor'));
-  task('dev', series('clean', 'copy', 'sass', 'webpack-dev-server'));
+  task('update-api', series('clean', 'copy', 'ts', 'update-api-extractor'));
+  task('dev', series('clean', 'copy', 'webpack-dev-server'));
 
   task('build:node-lib', series('clean', 'copy', series(condition('validate', () => !argv().min), 'ts:commonjs-only'))).cached();
 
-  task('bundle', argv().useMetro ? 'metroPack' : 'webpack').cached();
+  task('bundle', series(condition('metroPack', () => argv().useMetro), condition('webpack', () => !argv().useMetro))).cached();
```

Comment:
why this change?

Reply:
It turns out that using argv() directly in just.config.js (not inside a function) wreaks havoc with the just tasks.  It is somewhat indeterminate what happens but generally it does things like causes tasks to disappear.  I talked to Ken about this and since just doesn't have a ternary operator he recommended this series with two conditions type of approach.